sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/SampleManagementSteps.java

```diff
@@ -67,7 +67,7 @@ public SampleManagementSteps(
           int maximumNumberOfRows = 23;
           webDriverHelpers.waitUntilNumberOfElementsIsExactlyOrLess(
               SEARCH_RESULT_SAMPLE, maximumNumberOfRows);
-          Thread.sleep(1000); //reset filter acts chaotic, to be modified in the future
+          Thread.sleep(1000); // reset filter acts chaotic, to be modified in the future
```

Comment:
please add a //TO DO label  above thread sleep, we'll be able to scan it with intelliJ and fix it after

Reply:
this was not added by me it was just formatted maybe @VGitVlad knows more about it since he added it